src/backend/metal/src/command.rs

```diff
@@ -2764,21 +2880,21 @@ impl com::RawCommandBuffer<Backend> for CommandBuffer {
 
                 compute_commands.push(soft::ComputeCommand::BindBuffer {
                     index: 0,
-                    buffer: Some(dst.raw.clone()),
+                    buffer: Some(&*dst.raw),
                     offset: r.dst,
                 });
                 compute_commands.push(soft::ComputeCommand::BindBuffer {
                     index: 1,
-                    buffer: Some(src.raw.clone()),
+                    buffer: Some(&*src.raw),
                     offset: r.src,
                 });
                 compute_commands.push(soft::ComputeCommand::BindBufferData {
                     index: 2,
-                    bytes: unsafe {
+                    words: unsafe {
                         slice::from_raw_parts(
-                            &(r.size as u32) as *const u32 as _,
-                            mem::size_of::<u32>()
-                        ).to_owned()
+                            &(r.size as u32) as *const u32,
+                            1,
```

Comment:
nit: `mem::size_of::<u32>() / WORD_SIZE` if we ever change this (as mentioned in other comment)